installer/linux/agent-upgrade.sh

```diff
@@ -19,7 +19,18 @@ if [ "${os}" = "linux" ]; then
     mv /opt/openbas-agent/openbas-agent_upgrade /opt/openbas-agent/openbas-agent
     chmod 755 /opt/openbas-agent/openbas-agent
 
-    echo "02. Restarting the service"
+    echo "02. Updating OpenBAS configuration file"
+    cat > /opt/openbas-agent/openbas-agent-config.toml <<EOF
```

Comment:
I am not sure it's a good way to do it.
If you modify your config file before the upgrade, you lost your changes.

Reply:
I am not sure to understand sorry, I did like the installation script but we can discuss together if you want.